### ISSUE TITLE
T/o Kerning 

### ISSUE THREAD

**Issue by davelab6** [TypeNetwork/Roboto-Extremo#85]
https://twitter.com/justvanrossum/status/811481272333778944?s=19

https://twitter.com/justvanrossum/status/811642464108806144?s=19

How is this dealt with in Extremo?

**Comment by dberlow** [comment]
This was presented to TN tool development in 2016, in the more complex form of Ti, which contains a multi-morph, based on weight, width, optical size and glyph substitution from Ti to T[dotlessi]. 

Font Bureau’s plan is to gather up, in making variable fonts, the complex of typographic requirements at the intersection of glyph architecture, harfbuzz, opentype layout, variables and user interfaces, for Latin, Cyrillic and Greek, to the combinatorial requirements of typical a serif and serif italic, and typical sans and slanted sans design classes.

We have been looking for tools to accomplish this, and if you know of any under development, we’d be interested in discussing the functional and UI specification. Other than that, we are planning on having lots of informal conversations on our github repositories, where anyone who feels like contributing to broad-ranging example-making being done in public, here at Extremo and at the Amstelvar repo.

Also relevant to this is weather report on extreme hours to be built to the same standards as Roberto classic, And whether we should just remove all the accented characters and do them to a lower quality standard.

> On Mar 1, 2020, at 6:13 PM, Dave Crossland <notifications@github.com> wrote:
> 

﻿
https://twitter.com/justvanrossum/status/811481272333778944?s=19

https://twitter.com/justvanrossum/status/811642464108806144?s=19

How is this dealt with in Extremo?

**Comment by davelab6** [comment]
I agree that there have probably not ever been made any font editor features or extensions to adequately empower the type designer to completely address this. 

Extremo should be built to exceed the quality standards of Roboto Classic, and removing all the accented characters nor doing them to a lower quality standard is good.
